skeleton/Skeleton.cpp

```diff
@@ -17,9 +17,10 @@ namespace {
     virtual bool runOnFunction(Function &F) {
       // Get the function to call from our runtime library.
       LLVMContext &Ctx = F.getContext();
-      Constant *logFunc = F.getParent()->getOrInsertFunction(
-        "logop", Type::getVoidTy(Ctx), Type::getInt32Ty(Ctx), NULL
-      );
+      std::vector<Type*> paramsType = {Type::getInt32Ty(Ctx)};
```

Comment:
A better name here is probably `paramTypes`, since there are multiple types. (The current name implies that it’s a single type.)